styles/App.scss

```diff
@@ -22,6 +22,9 @@
 @import 'components/SocialMedia';
 @import 'components/TechGala/ProjectCard';
 @import 'components/TechGala/ViewBanner';
+@import 'components/Internship/CommitteeInternshipInfo';
+@import 'components/Internship/InternshipTimeline';
+@import 'components/Internship/NextSteps';
```

Comment:
Can we module these instead? For new CSS, unless there's a good reason let's module the files so we don't have to deal with it *again* in #191.